zpay32/invoice.go

```diff
@@ -290,7 +290,9 @@ func Decode(invoice string, net *chaincfg.Params) (*Invoice, error) {
 	// The next characters should be a valid prefix for a segwit BIP173
 	// address that match the active network.
 	if !strings.HasPrefix(hrp[2:], net.Bech32HRPSegwit) {
-		return nil, fmt.Errorf("unknown network")
+		return nil, fmt.Errorf(
+			"\"%s\" is the active network, but the specified invoice " +
```

Comment:
suggestion: "Invoice not for current active network '%s'"

Reply:
Done.

I used a lower case "invoice" in order to be consistent with the initial lower casing of other error message is that file.